### PR TITLE
RDAPI-21214 - Insecure Transport Update

### PR DESCRIPTION
**Pull request by cgardineraxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Customer who have proxy servers with HTTPS can now request WSDL API's securely over HTTPS by adding a VMArg to the jvm.xml file.

## Deploy preview link

https://deploy-preview-1836--axway-open-docs.netlify.app/docs/apim_administration/apimgr_admin/api_mgmt_config/#update-the-jvm-settings

https://deploy-preview-1836--axway-open-docs.netlify.app/docs/apim_reference/system_props/#77-may-2021

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by clivequinn** [comment]
Good stuff @cgardineraxway  - i think we will need one more piece of data in here. We need to tell customers that in order for the HTTPS proxy to work, the customer will need to import the proxy server cert into the API Gateway's certificate store.

**Comment by cgardineraxway** [comment]
> Good stuff @cgardineraxway - i think we will need one more piece of data in here. We need to tell customers that in order for the HTTPS proxy to work, the customer will need to import the proxy server cert into the API Gateway's certificate store.

Thanks @clivequinn, hopefully the new update makes sense. If not I can amend with any new suggestions. Thanks!

**Comment by dmchughaxway** [comment]
Hi @cgardineraxway, 

Can I just ask a question about the documentation. It says that 'The following JVM setting is also required when importing the API in API Manager.' That implies to me that that JVM setting is required for ALL imports - WSDL and API. However, the piece of code you fixed was only called when you imported WSDL right? My question is, does the paragraph that follows (about the certs) hold true even if you are importing a REST API and not a SOAP service?

Thanks,

Deirdre

**Comment by cgardineraxway** [comment]
@markduig-ax mentioned that the following page needs to be updated https://axway-open-docs.netlify.app/docs/apim_reference/system_props/ with any new system properties. As we have a new one here for http.proxyScheme we need to add this to that page.

**Comment by cgardineraxway** [comment]
> Hi @cgardineraxway,
> 
> Can I just ask a question about the documentation. It says that 'The following JVM setting is also required when importing the API in API Manager.' That implies to me that that JVM setting is required for ALL imports - WSDL and API. However, the piece of code you fixed was only called when you imported WSDL right? My question is, does the paragraph that follows (about the certs) hold true even if you are importing a REST API and not a SOAP service?
> 
> Thanks,
> 
> Deirdre

@dmchughaxway 

The updated code gets hit if we are retrieving an API from a URL. This is the same for swagger 2, swagger 3, WSDL and WADL.

**Comment by clivequinn** [comment]
Looks good @cgardineraxway

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_config.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ The following JVM setting is also required when importing the API in API Manage
...
+If the proxy server is capable of receiving traffic over HTTPS, it is recommended to add a VMArg to send the download request securely over HTTPS, for example, ```<VMArg name="-Dhttp.proxyScheme=https" />```. In addition to the VMArg, you should also add the server cert of the proxy server to the API Gateway's trusted certificate store, which can be done in Policy Studio, see [Manage X.509 certificates and keys](/docs/apim_administration/apigtw_admin/general_certificates). The download request is sent over HTTP if `http.proxyScheme` is set to `http` or if the `http.proxyScheme VMArg` is omitted from the jvm.xml file.
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apimgr_admin/api_mgmt_config.md:97]: @cgardineraxway I've made just a few small changes here. Are they ok? Can I merge this PR?

Comment (cgardineraxway) [reply content/en/docs/apim_administration/apimgr_admin/api_mgmt_config.md:97]: Hey @andreamussap . Hey that changes are good. Thanks for that. I think we are OK to merge.

**Review by andreamussap — APPROVED** [review]

**Pull request merged**